monkey/monkey_island/cc/resources/zero_trust/zero_trust_report.py

```diff
@@ -0,0 +1,38 @@
+import http.client
+
+import flask_restful
+from flask import Response, jsonify
+
+from monkey_island.cc.models.zero_trust.scoutsuite_data_json import ScoutSuiteDataJson
+from monkey_island.cc.resources.auth.auth import jwt_required
+from monkey_island.cc.services.zero_trust.zero_trust_finding_service import ZeroTrustFindingService
+from monkey_island.cc.services.zero_trust.zero_trust_service import ZeroTrustService
+
+REPORT_DATA_PILLARS = "pillars"
+REPORT_DATA_FINDINGS = "findings"
+REPORT_DATA_PRINCIPLES_STATUS = "principles"
+REPORT_DATA_SCOUTSUITE = "scoutsuite"
+
+
+class ZeroTrustReport(flask_restful.Resource):
+
+    @jwt_required
+    def get(self, report_data=None):
+        if report_data == REPORT_DATA_PILLARS:
+            return jsonify({
+                "statusesToPillars": ZeroTrustService.get_statuses_to_pillars(),
+                "pillarsToStatuses": ZeroTrustService.get_pillars_to_statuses(),
+                "grades": ZeroTrustService.get_pillars_grades()
+            })
+        elif report_data == REPORT_DATA_PRINCIPLES_STATUS:
+            return jsonify(ZeroTrustService.get_principles_status())
+        elif report_data == REPORT_DATA_FINDINGS:
+            return jsonify(ZeroTrustFindingService.get_all_findings())
+        elif report_data == REPORT_DATA_SCOUTSUITE:
+            try:
+                data = ScoutSuiteDataJson.objects.get().scoutsuite_data
+            except Exception:
+                data = "{}"
+            return Response(data, mimetype='application/json')
```

Comment:
Because data is already json

Reply:
I understand, but it seems weird you'll need a more complex statement for this case.
If you can solve this somehow that'll be best, but it's very minor, so I'm willing to skip it

Reply:
Added a comment. It's not weird because it's not a more complex statement. `jsonify` also forms a `Response` with mimetype and all that, except it also encodes the contents, which we don't need.